### DISCUSSION TITLE
Translate language does not work

### DISCUSSION THREAD

**Discussion by Nopaxa** [mattbanks/woocommerce-local-pickup-time#209] [Q&A]
Hi,
I try to adapt the texts via Loco Translate, but the translation is not accepted or displayed. Does anyone have any tips on how to get the text translated into another language?

**Comment by timnolte** [comment]
Hmm, the plugin has historically supported translations, and there are a number submitted on translate.wordpress.org so I'm pretty sure it should be working. Can you provide an example URL or some screenshots of where this isn't working?

**Comment by Nopaxa** [comment]
Hello Tim,
First of all I want to thank you for the plugin and your work on it.

I am attaching screenshots:

<img width="1015" alt="Bildschirmfoto 2023-04-07 um 15 12 08" src="https://user-images.githubusercontent.com/43169231/230615749-5475acb7-7bac-4476-a15c-a44aae2ef66e.png">
<img width="432" alt="Bildschirmfoto 2023-04-07 um 15 13 09" src="https://user-images.githubusercontent.com/43169231/230615783-5e5717b7-5a5b-413f-acf6-2e6b8bb75773.png">
<img width="612" alt="Bildschirmfoto 2023-04-07 um 20 08 34" src="https://user-images.githubusercontent.com/43169231/230656935-d535fd9a-d66d-4442-8e30-81b6d7a70453.png">

URL: wallboxcenter.de

**Comment by Nopaxa** [comment]
I have now done more translations on Loco and found that the translation works for the most part in the plugin settings.
